_docs/reference/cli/logs.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: guide
+title: StorageOS Docs - Logs
+anchor: reference
+module: reference/cli/logs
+---
+
+# Logs
+
+```bash
+Usage:	storageos logs COMMAND
+
+View and manage node logs
+
+Options:
+      --clear-filter       Clears the filter
+      --filter string      Set the logging filter
+  -f, --follow             Tail the logs for the given node, or all nodes if not specified
+      --help               Print usage
+  -t, --timeout int        Timeout in seconds. (default 5)
+      --verbosity string   Set the logging verbosity
+
+Commands:
+  view        Show logging configuration
+
+Run 'storageos logs COMMAND --help' for more information on a command.
+```
+
+The `logs` command is intended to assist with troubleshooting a running cluster.
+
+Verbosity controls the level of detail shown in the logs, with `info` the
+default.  `debug`, `info`, `warning`, `error` and `fatal` available.  During
```

Comment:
`debug`, `info`, `warning`, `error` and `fatal` available.
=>
The available options are `debug`, `info`, `warning`, `error` and `fatal`.